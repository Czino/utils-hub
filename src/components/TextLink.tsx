export const TextLink = ({ className, children, ...props }: React.ComponentProps<'a'>) => (
  <a
    className={['text-highlight-1 hover:underline active:text-highlight-2 visited:text-success-2', className].join(' ')}
    {...props}
  >
    {children}
  </a>
)
