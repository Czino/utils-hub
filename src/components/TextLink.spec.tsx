import { render } from '@testing-library/react'
import { TextLink } from './TextLink'

describe('TextLink', () => {
  it('should render TextLink', () => {
    const { asFragment } = render(
      <TextLink className="mt-4" href="/test">
        Test
      </TextLink>,
    )
    expect(asFragment()).toMatchSnapshot()
  })
})
