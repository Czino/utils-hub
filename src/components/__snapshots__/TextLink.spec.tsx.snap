// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextLink should render TextLink 1`] = `
<DocumentFragment>
  <a
    class="text-highlight-1 hover:underline active:text-highlight-2 visited:text-success-2 mt-4"
    href="/test"
  >
    Test
  </a>
</DocumentFragment>
`;
